### ISSUE TITLE
Review Codes of Conduct

### ISSUE THREAD

**Issue by jeff-zucker** [solid/deit#6]
We should take a look at the [Solid code of conduct](https://github.com/solid/process/blob/main/code-of-conduct.md) to see if we want to emphasize anything inclusion related and to clarify what role DEIT can play.  

I'd also suggest that we do as the spec team has done and explicitly state that we also adhere to the [W3C Code of Ethincs & Professional Conduct (Positive Work Environment)](https://www.w3.org/Consortium/cepc/).  It has much more specific language, for example labeling misgendering and deadnaming as forms of harassment.

**Comment by aschrijver** [comment]
We had some recent issues at [SocialHub](https://socialhub.activitypub.rocks) community that required us to revamp our Code of Conduct.

For the most part we could refer to the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/), but in addition to that we added a small text. This because we do not allow app developers or contributors who are involved with federated apps that are against what is in the W3C guidelines. E.g. when they are used for oppression or to promote extremist views, etc. (the title of this policy could be improved imho, and the text will be refined more in future policy versions).

https://socialhub.activitypub.rocks/t/socialhub-community-values-policy/1391

The nice thing of Discourse software is that you can reinforce this policy. As soon as members reach `trust_level_1` membership they _have to_ accept the policy, or they get a notification to do so on each visit. Once they formally accepted that becomes the basis on which we can moderate members if they cross the line.

By the way, we have a Well-being team for that. It consists of a forum group, and forum subcategory that is only visible to `trust_level_2` and above. The group ensures objective decision-making and can also judge e.g. mods and admins when they receive a flag. The subcategory ensures transparency, informing trusted members about moderation actions.

**Comment by jeff-zucker** [comment]
Yes, unfortunately, perhaps something like that Social Hub addendum is needed.

**Comment by Vinnl** [comment]
Codes of Conduct are a tricky thing, as they can also be weaponised as exclusionary tool - the exact opposite of what you'd want :) Of course my view is somewhat limited, but from my point of view, this could be helpful to make sure we get it right: https://files.frameshiftconsulting.com/codeofconducttraining.pdf

**Comment by jeff-zucker** [comment]
Nice resource @Vinnl.  I think it contains some good guidelines for how to handle violations - having a code of conduct is not worth anything unless there is an established procedure to handle violations through a code-of-conduct committee.  I guess we need to decide if the chairs of DEIT constitute that committee or if we need a separate named body.  In either case, that body needs to define a procedure for handling reports of misconduct.

**Comment by jeff-zucker** [comment]
Well, here's a bit of prehistory of DEI from a comment I made about Solid community code of conduct over two years ago " https://github.com/solid/information/pull/66#issuecomment-468083207

**Comment by timea-solid** [comment]
My review on the CoC:
- I see that the CoC is modeled structurally after https://www.w3.org/Consortium/cepc/ however I feel the https://www.womenofsolid.org/code-of-conduct.html easier to read and follow. 
On the topic of reporting:
- the Solid Manager as a sole decision power over an delicate issue makes me feel uncomfortable. I'd prefer to see a consortium with actual names of people in it (make it more personal and easier to approach). According to https://files.frameshiftconsulting.com/codeofconducttraining.pdf (who by the way I really really like and follow) there should be 3+ or more in such a committee. _Question_: could this be people from DEIT?
- a dedicated email for it like coc@solidproject.org would also make me feel more like 'this is really taken serious in this community'
Other: 
- I find actual examples about good and bad practices like on https://www.womenofsolid.org/code-of-conduct.html better, less vague
- Escalation examples and how it is going to be handled is also encouraging me more to want to come forward with an issue. Example also on: https://www.womenofsolid.org/code-of-conduct.html
